municipal_finance/static/javascript/table.js

```diff
@@ -687,6 +710,7 @@
 
       if (cube.columns.length > 1) blanks++;
       if (!_.isEmpty(this.filters.get('functions'))) blanks++;
+      if (CUBE_NAME == 'capital_v2') blanks++;
 
       for (var i = 0; i < blanks; i++) {
         var spacer = $('<th>').html('&nbsp;').addClass('spacer');
```

Comment:
Is there a specific reason why we'er setting all `th` to `&nbsp;` and not just empty?

Reply:
It seems to add a space to the `th`, `''` and `' '` result in the following
![image](https://user-images.githubusercontent.com/9511724/191331695-69059c86-d3ff-4fb2-a8e3-def065eee44e.png)